src/components/Banners/Banner.jsx

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types'
+import styles from '../Banners/HeroBanner.module.css'
+const Banner = ({url, title, main, addL1, addL2}) => {
+  return (
+    <div className={styles.HeroBanner}>
+      <img src={url} alt="Banner" className={styles.HeroBanner_img} />
+
+      <div className={styles.HeroBanner_info}>
+        <h2 className={styles.HeroBanner_add}>{title}</h2>
+        <h1 className={styles.HeroBanner_main}>{main}</h1>
+        <h2 className={styles.HeroBanner_add}>{addL1}</h2>
+        <h2 className={styles.HeroBanner_add}>{addL2}</h2>
+      </div>
+    </div>
```

Comment:
Wrzuć to w component Section

Reply:
Hej, nie bardzo wiem, jak to powinnam zrobić

Reply:
ten div class=HeroBanner służy jako sekcja więc zamiast tego zmień to w ten sposób że:
const Banner = ({url, title, main, addLine1, addLine2}) => {
  return (
    <Section
      sectionClass={css.HeroBanner}
      titleClass={css.HeroBannerTitle}
      title="Konferencja"
    >
      <img src={url} alt="Banner" className={styles.HeroBanner_img} />

      <div className={styles.HeroBanner_info}>
       <h2 className={styles.HeroBanner_add}>{title}</h2>
        <h1 className={styles.HeroBanner_main}>{main}</h1>
        <h2 className={styles.HeroBanner_add}>{addLine1}</h2>
        <h2 className={styles.HeroBanner_add}>{addLine2}</h2>
      </div>
    </Section>
  )
}
to oczywiscie podstawowa zmiana wtedy ta sekcje sobie ostyluj jak trzeba. mozesz podpatrzec sekcje Comments albo PreviousEdition to bedziesz miała podglad jak powrzucać poszczególne elementy

Reply:
Do tego te <h2>i <h1> bym zmienił
I jeszcze jedno pytanie. Ten img służy jako background?